assets/src/components/activities/creation.ts

```diff
@@ -20,7 +20,7 @@ export function registerCreationFunc(manifest: Manifest, fn: creationFn) {
     window.oliCreationFuncs = {};
   }
 
-  window.oliCreationFuncs[manifest.id] = fn;
+  window.oliCreationFuncs[manifest.authoring.element] = fn;
```

Comment:
Change the creation function key from the id to the authoring element to support multiple different LTI activities which will all have different IDs but all have the same authoring element. After all, this authoring element is actually what this creation functions map is used for.